src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Kiezkasse/Process/Module.ts

```diff
@@ -15,11 +10,7 @@ export var register = (angular) => {
     angular
         .module(moduleName, [
             AdhHttpModule.moduleName,
-            AdhMeinberlinPhaseModule.moduleName,
-            AdhMovingColumnsModule.moduleName,
-            AdhPermissionsModule.moduleName,
-            AdhTabsModule.moduleName,
-            AdhTopLevelStateModule.moduleName
+            AdhPermissionsModule.moduleName
```

Comment:
As far as I can see, `AdhMovingColumns` is actually needed, and also `AdhProcess` which was not in the list before.